src/wrappers/contract_wrappers/collateralizer_wrapper.ts

```diff
@@ -0,0 +1,482 @@
+            const self = this as CollateralizerContract;
+            const abiEncodedTransactionData = self.web3ContractInstance.seizeCollateral.getData();
+            return abiEncodedTransactionData;
+        },
+    };
+    public transferOwnership = {
+        async sendTransactionAsync(newOwner: string, txData: TxData = {}): Promise<string> {
+            const self = this as CollateralizerContract;
+            const txDataWithDefaults = await self.applyDefaultsToTxDataAsync(
+                txData,
+                self.transferOwnership.estimateGasAsync.bind(self, newOwner),
+            );
+            const txHash = await promisify<string>(
+                self.web3ContractInstance.transferOwnership,
+                self.web3ContractInstance,
+            )(newOwner, txDataWithDefaults);
+            return txHash;
+        },
+        async estimateGasAsync(newOwner: string, txData: TxData = {}): Promise<number> {
+            const self = this as CollateralizerContract;
+            const txDataWithDefaults = await self.applyDefaultsToTxDataAsync(txData);
+            const gas = await promisify<number>(
+                self.web3ContractInstance.transferOwnership.estimateGas,
+                self.web3ContractInstance,
+            )(newOwner, txDataWithDefaults);
+            return gas;
+        },
+        getABIEncodedTransactionData(newOwner: string, txData: TxData = {}): string {
+            const self = this as CollateralizerContract;
+            const abiEncodedTransactionData = self.web3ContractInstance.transferOwnership.getData();
+            return abiEncodedTransactionData;
+        },
+    };
+    public getAuthorizedCollateralizeAgents = {
+        async callAsync(defaultBlock?: Web3.BlockParam): Promise<string[]> {
+            const self = this as CollateralizerContract;
+            const result = await promisify<string[]>(
+                self.web3ContractInstance.getAuthorizedCollateralizeAgents.call,
+                self.web3ContractInstance,
+            )();
+            return result;
+        },
+    };
+
+    async deploy(...args: any[]): Promise<any> {
+        const wrapper = this;
+        const rejected = false;
+
+        return new Promise((resolve, reject) => {
+            wrapper.web3ContractInstance.new(
+                wrapper.defaults,
+                (err: string, contract: Web3.ContractInstance) => {
+                    if (err) {
+                        reject(err);
+                    } else if (contract.address) {
+                        wrapper.web3ContractInstance = wrapper.web3ContractInstance.at(
+                            contract.address,
+                        );
+                        wrapper.address = contract.address;
+                        resolve();
+                    }
+                },
+            );
+        });
+    }
+
+    static async deployed(web3: Web3, defaults: Partial<TxData>): Promise<CollateralizerContract> {
```

Comment:
This is a redundant declaration -- please remove this declaration (the one further down the file is accurate)